### DISCUSSION TITLE
Availability of the AVM public container registry

### DISCUSSION THREAD

**Discussion by Bascz** [Azure/Azure-Verified-Modules#1157] [Q&A]
We are currently in the process of converting our private building blocks to use Azure Verified Modules. One question that arises is what the SLA/availability is of the container registry where the modules are stored. Can anything be said about this?

**Comment by matt-FFFFFF** [comment]
@jtracey93 do you know the answer to this?

**Comment by jtracey93** [comment]
Hey @Bascz,

All I can share today is this blog post: https://azure.microsoft.com/en-us/blog/microsoft-syndicates-container-catalog/

I have reached out internally to see if there are any further details I can share.
